go.mod

```diff
@@ -10,8 +10,15 @@ require (
 	github.com/mitchellh/go-testing-interface v1.0.3 // indirect
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/stackdriverexporter v0.5.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/processor/metricstransformprocessor v0.6.1-0.20200723171718-a2ff1aa6779e
+	github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourcedetectionprocessor v0.6.0
```

Comment:
Note that you now depend on different versions of modules within `opentelemetry-collector-contrib` (see Stackdriver exporter and Metrics Transform Processor above). Would be nice to use a consistent version across all of them to avoid some implicit issue with the upstream dependency versions.

Reply:
Have updated them all to the latest version of Contrib